src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
Here's the thing though, if you have a slider with multiple handles then it doesn't really make sense to have a single prop. You'd pretty much have to pass in your own handle component.

I think the moral here is if you want to be accessible then you can't leave things to default.